build.gradle

```diff
@@ -37,45 +36,50 @@ allprojects {
     apply plugin: "org.jlleitschuh.gradle.ktlint"
 
     repositories {
-        // For olm library. This has to be declared first, to ensure that Olm library is not downloaded from another repo
-        maven { url 'https://gitlab.matrix.org/api/v4/projects/27/packages/maven' }
-
+        // For olm library.
+        maven {
+            url 'https://gitlab.matrix.org/api/v4/projects/27/packages/maven'
+            content {
+                groups.olm.regex.each { includeGroupByRegex it }
+                groups.olm.group.each { includeGroup it }
+            }
+        }
         maven {
             url 'https://jitpack.io'
             content {
-                // Use this repo only for FilePicker
-                includeGroupByRegex "com\\.github\\.jaiselrahman"
-                // And monarchy
-                includeGroupByRegex "com\\.github\\.Zhuinden"
-                // And ucrop
-                includeGroupByRegex "com\\.github\\.yalantis"
-                // JsonViewer
-                includeGroupByRegex 'com\\.github\\.BillCarsonFr'
-                // PhotoView
-                includeGroupByRegex 'com\\.github\\.chrisbanes'
-                // PFLockScreen-Android
-                includeGroupByRegex 'com\\.github\\.vector-im'
-                // DraggableView
-                includeGroupByRegex 'com\\.github\\.hyuwah'
-
-                // Chat effects
-                includeGroupByRegex 'com\\.github\\.jetradarmobile'
-                includeGroupByRegex 'nl\\.dionsegijn'
-
-                // Voice RecordView
-                includeGroupByRegex 'com\\.github\\.Armen101'
+                groups.jitpack.regex.each { includeGroupByRegex it }
+                groups.jitpack.group.each { includeGroup it }
             }
         }
-        maven { url 'https://oss.sonatype.org/content/repositories/snapshots/' }
```

Comment:
I am not sure why it has been added at first place, but it seems not necessary anymore 🤞